<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>nf-core/clinvap: Output</title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>





<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 13px;
}

body {
  max-width: 800px;
  margin: auto;
  padding: 1em;
  line-height: 20px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 {
   font-size:2.2em;
}

h2 {
   font-size:1.8em;
}

h3 {
   font-size:1.4em;
}

h4 {
   font-size:1.0em;
}

h5 {
   font-size:0.9em;
}

h6 {
   font-size:0.8em;
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre, img {
  max-width: 100%;
}
pre {
  overflow-x: auto;
}
pre code {
   display: block; padding: 0.5em;
}

code {
  font-size: 92%;
  border: 1px solid #ccc;
}

code[class] {
  background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * {
      background: transparent !important;
      color: black !important;
      filter:none !important;
      -ms-filter: none !important;
   }

   body {
      font-size:12pt;
      max-width:100%;
   }

   a, a:visited {
      text-decoration: underline;
   }

   hr {
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote {
      padding-right: 1em;
      page-break-inside: avoid;
   }

   tr, img {
      page-break-inside: avoid;
   }

   img {
      max-width: 100% !important;
   }

   @page :left {
      margin: 15mm 20mm 15mm 10mm;
   }

   @page :right {
      margin: 15mm 10mm 15mm 20mm;
   }

   p, h2, h3 {
      orphans: 3; widows: 3;
   }

   h2, h3 {
      page-break-after: avoid;
   }
}
 
body {
  padding: 3em;
  margin-right: 350px;
  max-width: 100%;
}
#toc {
  position: fixed;
  right: 20px;
  width: 300px;
  padding-top: 20px;
  overflow: scroll;
  height: calc(100% - 3em - 20px);
}
#toc_header {
  font-size: 1.8em;
  font-weight: bold;
}
#toc > ul {
  padding-left: 0;
  list-style-type: none;
}
#toc > ul ul { padding-left: 20px; }
#toc > ul > li > a { display: none; }
img { max-width: 800px; }

</style>



</head>

<body>
<div id="toc">
<div id="toc_header">Table of Contents</div>
<ul>
<li>
<a href="#toc_0">nf-core/clinvap: Output</a>
<ul>
<li>
<a href="#toc_1">Pipeline overview</a>
</li>
<li>
<a href="#toc_2">VEP</a>
</li>
<li>
<a href="#toc_3">Report Generation</a>
</li>
</ul>
</li>
</ul>
</div>


<h1 id="toc_0">nf-core/clinvap: Output</h1>

<p>This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.</p>

<!-- TODO nf-core: Write this documentation describing your workflow's output -->

<h2 id="toc_1">Pipeline overview</h2>

<p>The pipeline is built using <a href="https://www.nextflow.io/">Nextflow</a>
and processes data using the following steps:</p>

<ul>
<li><a href="#vep">VEP</a> - Variant effect prediction via Ensembl VEP</li>
<li><a href="#report">Report Generation</a> - Aggregate clinical variant annottion report</li>
</ul>

<h2 id="toc_2">VEP</h2>

<p><a href="https://www.ensembl.org/info/docs/tools/vep/script/vep_options.html">VEP</a> is used to annotate raw VCF files. The annotation includes the predicted effects of the mutations on the protein function.</p>

<p><strong>Output directory: <code>results</code></strong></p>

<ul>
<li><code>&lt;VCF&gt;.vcf</code>

<ul>
<li>Annotated VCF file</li>
</ul></li>
</ul>

<h2 id="toc_3">Report Generation</h2>

<p><a href="#report">Report Generation</a> processes the information found in the annotated VCF file and conducts further annotations including driver gene type and therapeutic suggestions.
The resulting json report includes 6 main categories:</p>

<ol>
<li>&quot;mskdg&quot;: Somatic Mutations in Known Driver genes<br>
List of cancer driver genes along with the observed mutations in the patient.</li>
<li>&quot;ptp_da&quot;: Summary of Cancer Drugs Targeting the Affected genes<br>
List of cancer drugs targeting the mutated gene.</li>
<li>&quot;ptp_ia&quot;: CIViC Summary of Drugs Targeting the Affected genes<br>
Therapies that have evidence of targeting the affected gene</li>
<li>&quot;mskpe&quot;: Somatic Mutations with Known Pharmacogenetic Effect
List of drugs that directly targets the observed variant of the gene<br></li>
<li>&quot;ref&quot;: References<br>
The publications as an evidence of the found annotation/information</li>
<li>&quot;appendix&quot;: Appendix<br>
All the somatic variants of the patient with their dbSNP and COSMIC IDs.</li>
</ol>

<p><strong>Output directory: <code>results</code></strong></p>

<ul>
<li><code>&lt;REPORT&gt;.json</code>

<ul>
<li>Clinical report in json format.</li>
</ul></li>
</ul>

</body>

</html>
